geaflow/geaflow-common/src/main/java/com/antgroup/geaflow/common/binary/BinaryString.java

```diff
@@ -226,6 +229,67 @@ public boolean contains(final BinaryString substring) {
         return false;
     }
 
+    public static BinaryString concat(BinaryString... inputs) {
+        long totalLength = 0;
+        for (BinaryString input : inputs) {
+            if (Objects.isNull(input)) {
+                continue;
+            }
+            totalLength += input.numBytes;
+        }
+
+        byte[] result = new byte[Math.toIntExact(totalLength)];
+        int offset = 0;
+        for (BinaryString input : inputs) {
+            if (Objects.isNull(input)) {
+                continue;
+            }
+            int len = input.numBytes;
+            copyMemory(input.binaryObject, input.offset, result, offset, len);
+            offset += len;
+        }
+        return fromBytes(result);
+    }
+
+    public static BinaryString concatWs(String separator, BinaryString... inputs) {
```

Comment:
The type for `separator`  should also be `BinaryString`

Reply:
done!